openff/toolkit/tests/test_forcefield.py

```diff
@@ -75,11 +75,6 @@
 # GLOBAL CONSTANTS
 # ======================================================================
 
-# File paths.
-TIP3P_SDF_FILE_PATH = get_data_file_path(
-    os.path.join("systems", "monomers", "water.sdf")
-)
```

Comment:
(not blocking) Oh, interesting. I guess this was never used?

Reply:
It looks like the file is used in a few places, but this shortcut to the file is not used at the moment. Maybe it was in the past - not sure, thought it better to remove it anyway.